benchmarks/distributed/rpc/rl/agent.py

```diff
@@ -0,0 +1,150 @@
+from functools import reduce
+import time
+import threading
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+
+import torch.distributed.rpc as rpc
+from torch.distributed.rpc import RRef, remote
+from torch.distributions import Categorical
+
+from torch import autograd
+
+OBSERVER_NAME = "observer{}"
+gamma = 0.99
+
+torch.autograd.set_detect_anomaly(True)
+
+
+class Policy(nn.Module):
+    def __init__(self, state_size, nlayers, out_features, batch=True):
```

Comment:
the `state_size` does not belong to policy. If you need to infer `in_features` from state_size, that needs to be done outside of Policy and Policy should take in_features, nlayers, out_features.